inc/console/database/updatecommand.class.php

```diff
@@ -0,0 +1,195 @@
+         return $db_error;
+      }
+
+      // Initialize entities
+      $_SESSION['glpidefault_entity'] = 0;
+      Session::initEntityProfiles(2);
+      Session::changeProfile(4);
+
+      // Display current/future state informations
+      $currents            = $update->getCurrents();
+      $current_version     = $currents['version'];
+      $current_db_version  = $currents['dbversion'];
+
+      global $migration;
+      $migration = new CliMigration(GLPI_SCHEMA_VERSION);
+      $migration->setOutput($output);
+      $update->setMigration($migration);
+
+      $informations = new Table($output);
+      $informations->setHeaders(['', __('Current'), __('Target')]);
+      $informations->addRow([__('GLPI version'), $current_version, GLPI_VERSION]);
+      $informations->addRow([__('GLPI database version'), $current_db_version, GLPI_SCHEMA_VERSION]);
+      $informations->render();
+
+      if (defined('GLPI_PREVER')) {
+         // Prevent unstable update unless explicitly asked
+         if (!$allow_unstable && version_compare($current_db_version, GLPI_SCHEMA_VERSION, 'ne')) {
+            $output->writeln(
+               sprintf(
+                  '<error>' . __('%s is not a stable release. Please upgrade manually or add --allow-unstable option.') . '</error>',
+                  GLPI_SCHEMA_VERSION
+               )
+            );
+            return self::ERROR_NO_UNSTABLE_UPDATE;
+         }
+      }
+
+      if (version_compare($current_db_version, GLPI_SCHEMA_VERSION, 'eq') && !$force) {
+         $output->writeln('<comment>' . __('No migration needed.') . '</comment>');
+         return 0;
+      }
+
+      if (!$no_interaction) {
+         // Ask for confirmation (unless --no-interaction)
+         /** @var QuestionHelper $question_helper */
+         $question_helper = $this->getHelper('question');
+         $run = $question_helper->ask(
+            $input,
+            $output,
+            new ConfirmationQuestion(__('Do you want to run continue ? [Y/n]'), true)
+         );
+         if (!$run) {
+            $output->writeln('<comment>' . __('Update aborted.') . '</comment>');
+            return 0;
+         }
+      }
+
+      if (substr($current_version, -4) === '-dev') {
+         // Normalize version
+         $current_version = str_replace('-dev', '', $current_version);
+      }
+
+      $update->doUpdates($current_version);
+
+      if (version_compare($current_db_version, GLPI_SCHEMA_VERSION, 'ne')) {
+         // Migration is considered as done as Update class has the responsibility
+         // to run updates if schema has changed (even for "pre-versions".
+         $output->writeln('<comment>' . __('Migration done.') . '</comment>');
+      } else if ($force) {
+         // Replay last update script even if there is no schema change.
+         // It can be used in dev environment when update script has been updated/fixed.
+         include_once(GLPI_ROOT . '/install/update_93_94.php');
```

Comment:
Out of the scope of the current PR, but if wa can find a way to get rid of those lines to change manually on each update :(

Reply:
Maybe in another PR. It could also be a good idea to remove `case GLPI_PREVER:` in classic update process and move it into a `Update::replayLast()` method or something like that.